client/src/components/HostInfo.js

```diff
@@ -34,56 +34,56 @@ class HostInfo extends React.Component {
       let pathname = window.location.pathname.split('/').pop();
       if (pathname === undefined || pathname === 'blank') {
         this.getHostInfoById(1);
-        this.getProfilePicture(1);
+        // this.getProfilePicture(1);
       } else {
         this.getHostInfoById(pathname);
-        this.getReviewCount(pathname);
-        this.getProfilePicture(pathname);
+        // this.getReviewCount(pathname);
+        // this.getProfilePicture(pathname);
         this.setState({
           id: Number(pathname)
         });
       }
     } else {
       let queryStringRes = queryString.split('?').pop();
       this.getHostInfoById(queryStringRes);
-      this.getReviewCount(queryStringRes);
-      this.getProfilePicture(queryStringRes);
+      // this.getReviewCount(queryStringRes);
+      // this.getProfilePicture(queryStringRes);
       this.setState({
         id: Number(queryStringRes)
       });
     }
   }
 
-  getReviewCount(id) {
-    http.get('https://fec-gai-hostprofile.s3-us-west-1.amazonaws.com/json/reviews.json')
-      .then(response => {
-          let reviewCount = response.data.filter((ele, i, self) => {
-            return ele.id === Number(id)
-          });
-          this.setState({
-            reviewCount: reviewCount[0].review_count
-          });
-      })
-      .catch(err => {
-        console.log('Error receiving review counts from AWS s3: ');
-      });
-  }
+  // getReviewCount(id) {
+  //   http.get('https://fec-gai-hostprofile.s3-us-west-1.amazonaws.com/json/reviews.json')
+  //     .then(response => {
+  //         let reviewCount = response.data.filter((ele, i, self) => {
+  //           return ele.id === Number(id)
+  //         });
+  //         this.setState({
+  //           reviewCount: reviewCount[0].review_count
+  //         });
+  //     })
+  //     .catch(err => {
+  //       console.log('Error receiving review counts from AWS s3: ');
+  //     });
+  // }
```

Comment:
I wonder why you commented out this code?

Reply:
That should have been part of the second module but was never implemented. It's not part of my db, it was coming from an S3 bucket, so I set some defaults instead.